src/utils/log.ts

```diff
@@ -1,57 +1,64 @@
-import { stderr as chalkStderr } from "chalk";
-import { CliKintoneError } from "./error";
+import winston from "winston";
 
-const currentISOString = () => new Date().toISOString();
+const { combine, timestamp, printf, splat } = winston.format;
 
-export type Logger = {
-  debug: (message: any) => void;
-  info: (message: any) => void;
-  warn: (message: any) => void;
-  error: (message: any) => void;
-  fatal: (message: any) => void;
-};
+export type LogLevel = "debug" | "info" | "warn" | "error" | "fatal";
+export type Logger = winston.Logger &
```

Comment:
Please keep having our `Logger` interface, not using Winston's interface directly.
Our logger should have an interface that is suitable for our domain.
If we use winston as our interface, it makes it difficult to change our logger, and changes in winston will affect our code directly.

Please use Winston as our logger backend.